test/functional/specs/Personalization/C28755.js

```diff
@@ -34,10 +34,10 @@ test.meta({
   TEST_RUN: "Regression"
 });
 
-test.skip("Test C28755: A VEC offer for all visitors should return in every event if __view__ scope exist", async () => {
+test("Test C28755: A VEC offer for all visitors should return in every event if __view__ scope exist", async () => {
```

Comment:
I think we should change the test case name to something like "The first sendEvent on the page should fetch Personalization VEC offers" or the test doesn't reflect the testcase name.
As we currently support SPA as well, I would validate in this testcase that the first sendEvent on the page would fetch Personalization by having the `__view__` scope in the request